tutorials/shading/shading_reference/shading_language.rst

```diff
@@ -514,9 +514,9 @@ When vec_type (float), vec_int_type, vec_uint_type, vec_bool_type nomenclature i
 +-----------------------------------------------------------------------------------------------+------------------------------------------------+
 | vec_type **mix** ( vec_type a, vec_type b, vec_bool_type c )                                  | Linear Interpolate (Bool-Vector Selection)     |
 +-----------------------------------------------------------------------------------------------+------------------------------------------------+
-| vec_type **step** ( vec_type a, vec_type b )                                                  | \` b[i] < a[i] ? 0.0 : 1.0 \`                  |
+| vec_type **step** ( vec_type a, vec_type b )                                                  | \`\`b[i] < a[i] ? 0.0 : 1.0\`\`            |
```

Comment:
This should be using two back ticks to make this inline code, I don't know why it was escaped initially. Using the backslashes makes it display the back ticks, which is not the desired look.